lib/srv/statichostusers.go

```diff
@@ -142,7 +151,7 @@ func (s *StaticHostUserHandler) run(ctx context.Context) error {
 			return trace.Errorf("missing init event from watcher")
 		}
 		s.retry.Reset()
-	case <-time.After(staticHostUserWatcherTimeout):
+	case <-s.clock.After(staticHostUserWatcherTimeout):
```

Comment:
This should probably be a timer that we explicitly stop if it's using a `clockwork.Clock`.